charts/grafana/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.50.7
+version: 6.50.8
```

Since this PR is a new feature, please bump to minor version.
```suggestion
version: 6.51.0
```